### PR TITLE
using newer dependencies (wreck and fast-safe-stringify)

### PR DESCRIPTION
**Pull request by doron2402**
@arb ^^

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "good-http",
-  "version": "6.1.1",
+  "version": "6.1.2",
```
Comment (arb) [inline package.json]: Undo this. The lead maintainer handles version changes.

File: package.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
     "test-cov-html": "lab -m 5000 -r html -o coverage.html -La code"
   },
   "dependencies": {
-    "fast-safe-stringify": "1.1.0",
-    "wreck": "7.x.x"
+    "fast-safe-stringify": "1.1.3",
+    "wreck": "10.0.0"
```
Comment (arb) [inline package.json]: For hapi dependencies, we use `#.x.x` since they are things "we" control.

Comment (doron2402) [inline package.json]: got it

File: package.json (hunk 1 above)
Comment (doron2402) [inline package.json]: Thanks

**Comment by doron2402** [comment]
hi @arb I've rebase my changes.

Thanks

**Pull request merged**